docs/reference/snapshot-restore/apis/restore-snapshot-api.asciidoc

```diff
@@ -106,8 +106,8 @@ include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=master-timeout]
 
 `wait_for_completion`::
 (Optional, Boolean) If `true`, the request returns a response when the restore
-operation completes. The operation completes once it attempts all
-<<_monitoring_restore_operations,recovery and replication steps>>.
+operation completes. A restore operation is considered complete once it attempts
+all <<_monitoring_restore_operations,recovery and replication steps>>.
```

Comment:
I think that's not quite true, it completes once all the primaries are there but it doesn't wait for replicas. I think we want to say something like this, although I imagine there's better ways to say it:

> A restore operation is considered complete after it has attempted to recover each primary shard from the snapshot and each of those attempts has finished, whether successfully or not.

Reply:
Thanks for the correction. I've fixed this with e691815.